src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/CreateFilteringFiles.java

```diff
@@ -38,6 +38,8 @@ public final class CreateFilteringFiles extends VariantWalker {
 
     private List<String> HEADER = 
         Arrays.asList("filter_set_name","mode","location","ref","alt","vqslod","culprit","training_label","yng");
+    private List<String> HEADER_VQSR_LITE =
```

Comment:
conceptually this is HEADER + "calibration_sensitivity" again, here and below

Reply:
see above.  It's only HEADER + calibration_sensitivity for the moment, but will not be after cleanup